load-test/send_load.sh

```diff
@@ -70,9 +74,7 @@ do
 done
 
 sleep 7s
-balance1=$("${DAEMON}" q bank balances "${acc1}" --node $RPC --output json)
-balance1res=$(echo "${balance1}" | jq -r '.balances')
-echo "** Balance of Account 1 after send_load :: $balance1res **"
-balance2=$("${DAEMON}" q bank balances "${acc2}" --node $RPC --output json)
-balance2res=$(echo "${balance2}" | jq -r '.balances')
-echo "** Balance of Account 2 after send_load :: $balance2res **"
\ No newline at end of file
+echo "** Balance of Account 1 after send_load :: **"
+balace_query $acc1
+echo "** Balance of Account 2 after send_load ::  **"
+balace_query $acc2
```

Comment:
Should we add `expected_balance` and `actual_balance` for comparison?

Reply:
Before and after balance query for this script is mainly for making sure there is change in balance after execution as we are not querying the individual tx hashes anymore. Balance check is necessary for `query_load.sh` script which is being tracked in this issue:- https://github.com/cosmos/qa-integration/issues/9